pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[tool.poetry]
+name = "caracal"
+version = "1.1.0"
+description = "Containerized Automated Radio Astronomy Calibration"
+authors = ["The CaraCAL Team <caracal-info@googlegroups.com>"]
+readme = "README.md"
+homepage = "https://github.com/caracal-pipeline/caracal"
+repository = "https://github.com/caracal-pipeline/caracal"
+keywords = ["Astronomy", "Visualisation", "Pipelines"]
+classifiers = [
+          "Development Status :: 5 - Production/Stable",
+          "Intended Audience :: Science/Research",
+          "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
+          "Operating System :: POSIX :: Linux",
+          "Programming Language :: Python",
+          "Topic :: Scientific/Engineering :: Astronomy"
+]
+license = "GNU GPL v2"
+packages = [
+    {include = "caracal"},
+]
+
+[tool.poetry.scripts]
+shadems = {reference = "bin/caracal", type = "file"}
+
+[tool.poetry.dependencies]
+python = "^3.8"
+stimela = "^1.7.6"
+python-casacore = "^3.5.2"
+psutil = "^5.9.4"
+jinja2 = "^3.1.2"
+radiopadre-client = "^1.2.0"
+nbconvert = "^7.2.10"
+numpy = "^1.24.2"
+matplotlib = "^3.6.0"
+regions = "^0.5"
+astropy = "^5.2.1"
+pykwalify = "^1.8.0"
+progressbar2 = "^4.2.0"
+pysolr = "^3.9.0"
+decorator = "^4.2.2"
+"ruamel.yaml" = "^0.17.21"
+scipy = "^1.9.3"
+pytest = { version = "^7.2.2", optional=true }
+pytest-flake8 = { version = "^1.1.1", optional=true }
```

Comment:
Given that caracal is an application, it might be useful to commit the poetry lock file, so that the dependencies are fixed for a specific version of caracal.